### PR TITLE
Subgraph V1.7.0 Release

### LINKED ISSUE #1814: [SUBGRAPH] add version (set at deployment) to sfmetas entity
This is needed for https://github.com/superfluid-finance/garden-infra/issues/84

We want subgraph_package_version and commit_hash in sfMeta.

### PR DESCRIPTION
**Pull request by 0xdavinchee**

### CODE REVIEW

File: packages/subgraph/tasks/getSFMeta.sh [hunk 1]
```diff
@@ -1,8 +1,12 @@
...
+# This is used to fill out the SFMeta entity (see schema.graphql)
+# The different fields: `${SOME_FIELD}` must be set in the enivronment when running this script.
 #!/usr/bin/env bash
 BRANCH="$(git branch --show-current)"
 
 cat > src/meta.ignore.ts << EOF
 export let commitHash = "${COMMIT_HASH}";
 export let configuration = "${CONFIGURATION}";
 export let branch = "${BRANCH}";
+export let packageVersion = "${PACKAGE_VERSION}";
```
Comment (hellwolf) [inline packages/subgraph/tasks/getSFMeta.sh:15]: where is PACKAGE_VERSION coming from?

**Comment by hellwolf** [comment]
Triggered a test: https://github.com/superfluid-finance/protocol-monorepo/actions/runs/7783183618

To be deployed: https://thegraph.com/hosted-service/subgraph/superfluid-finance/protocol-feature-mumbai

**Review by hellwolf — CHANGES_REQUESTED** [review]
bump subgraph version, and update changelog

File: packages/subgraph/CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,258 @@
+# Changelog
+
+All notable changes to the Subgraph will be documented in this file.
+
+This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+## [1.7.0] - 2024-02-05
```
Comment (hellwolf) [inline packages/subgraph/CHANGELOG.md:9]: GDA features are from 1.7.0 right?

Comment (0xdavinchee) [reply packages/subgraph/CHANGELOG.md:9]: yup

Comment (hellwolf) [reply packages/subgraph/CHANGELOG.md:9]: shouldn't we then write a few texts about it?

Comment (0xdavinchee) [reply packages/subgraph/CHANGELOG.md:9]: forgot to hit save after writing that part haha 🥲

**Comment by hellwolf** [comment]
Triggered a new deployment: https://github.com/superfluid-finance/protocol-monorepo/actions/runs/7787826966

Previous one was of wrong commit.

**Comment by hellwolf** [comment]
@elvijsTDL it's still syncing, but we can already test: https://api.thegraph.com/subgraphs/id/QmV4MZzeQSBDEqfzc91ESA43YCwW3m6q7NzaTKHYZsrwdm/graphql?query=query+MyQuery+%7B%0A++sfmetas+%7B%0A++++blockNumber%0A++++branch%0A++++configuration%0A++++id%0A++++packageVersion%0A++++timestamp%0A++%7D%0A%7D


image

**Review by hellwolf — APPROVED** [review]

**Pull request merged**